Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -6396,42 +6396,61 @@ def com_google_fonts_check_metadata_category_hint(family_metadata):
     id = "com.google.fonts/check/colorfont_tables",
     rationale = """
         Colr v0 fonts are widely supported in most browsers so they do not require
-        an SVG color table. However, Colr v1 is only well supported in Chrome so
-        we need to add an SVG table to these fonts.
+        an SVG color table. However, Colr v1 is currently not broadly supported
+        so we need to add an SVG table to these fonts, except in the case of
+        variable fonts, since SVG does not support OpenType Variations.
 
-        To add an SVG table, run the maximum_color tool in Nano Emoji,
+        To add an SVG table, run the maximum_color tool in nanoemoji:
```

Comment:
> To add an SVG table from a font with COLR (v0 or v1) table, or to add a COLRv1 table from a font with SVG table

Reply:
To automatically generate compatible SVG/COLR tables, run the maximum_color tool in nanoemoji